be/src/main/java/issuetracker/be/controller/IssueController.java

```diff
@@ -0,0 +1,27 @@
+package issuetracker.be.controller;
+
+import issuetracker.be.dto.IssueSaveRequestDto;
+import issuetracker.be.service.IssueService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class IssueController {
+
+  private IssueService issueService;
+
+  @Autowired
+  public IssueController(IssueService issueService) {
+    this.issueService = issueService;
+  }
+
+  @PostMapping("/issue")
+  public ResponseEntity<?> saveIssue(@RequestBody IssueSaveRequestDto issueSaveRequestDto) {
+    issueService.save(issueSaveRequestDto);
+    return ResponseEntity.status(HttpStatus.CREATED).build();
+  }
```

Comment:
- ? 와 void 는 다른 가치를 가집니다.

`nit` 
```suggestion
    @ResponseStatus(HttpStatus.CREATED)
    @PostMapping("/issue")
    public void saveIssue(@RequestBody IssueSaveRequestDto issueSaveRequestDto) {
        issueService.save(issueSaveRequestDto);
    }
```